tests/services/test_record_service.py

```diff
@@ -56,6 +48,26 @@ def test_create_draft(app, service, identity_simple, input_data):
     assert draft._record.conceptpid.status == PIDStatus.NEW
 
 
+def test_create_empty_draft(app, service, identity_simple):
+    """Test an empty draft can be created.
+
+    Errors (missing required fields) are reported, but don't prevent creation.
+    """
+    # Needs `app` context because of invenio_access/permissions.py#166
+    input_data = {
+        "metadata": {}
+    }
+
+    draft = service.create(identity_simple, input_data)
+    draft_dict = draft.to_dict()
+
+    assert draft['id']
+    assert draft['conceptid']
+    assert draft._record.pid.status == PIDStatus.NEW
+    assert draft._record.conceptpid.status == PIDStatus.NEW
+    assert draft_dict['errors']['metadata']['title']
```

Comment:
minor: what do you think about putting this in a different test, to make things more explicit.

Reply:
Added the check for no errors in the happy path create draft test instead.